src/CommunityVoices/App/Website/View/DisplayError.php

```diff
@@ -63,6 +63,35 @@ public function getError($request)
 
         $response = new HttpFoundation\Response($presentation->generate($domainXMLElement));
 
+        $this->finalize($response);
+        return $response;
+    }
+    public function getAllErrors($request) {
+
+        $errorsXMLElement = new SimpleXMLElement(
+            $this->transcriber->toXml(
+                $this->apiProvider->getJson('/errors', $request)
```

Comment:
Going off of other comments, I maybe like the path `/error-log`, but the route is no big deal.